index.html

```diff
@@ -523,8 +523,18 @@ <h2>
 
     <section>
       <h2>
-Keys
+Cryptographic Keys
```

Comment:
We should talk about "Verification Methods" somewhere and indicate that "Cryptographic Keys" are a subset.

Reply:
Changed section header to "Verification Methods" in 6d875bf5a55ff729700ec4f47a7301ecb1ad060c.

Reply:
Changed it back... the concept is so general that it's difficult to get across. Agree that we need it in there, but this PR is probably the wrong place to do it. Having a section called "Verification Methods" is confusing... "Cryptographic Keys" is an easier concept for people to grasp. Attempted to mention that "Cryptographic Keys" are one type of verification method... we need to provide other examples.

Reply:
@msporny @dlongley Maybe the section header can express this tension, and be "Cryptographic Keys and Other Verification Methods" ?

Reply:
Applied @dmitrizagidulin's title in 8aac222218dbe5114b5a41a03864c520a38693a0.